shared/models/videos/abuse/video-abuse-reason.model.ts

```diff
@@ -0,0 +1,21 @@
+export enum VideoAbusePredefinedReasons {
```

Comment:
You already defined this enum :thinking: 

I think you forgot to remove the one in `client/`?